packages/cloud2edge/post-install/hono-kafka-connection.json

```diff
@@ -151,8 +157,14 @@
         }
       ],
       "specificConfig": {
+        {{- if .Values.hono.kafkaMessagingClusterExample.enabled }}
         "saslMechanism": "plain",
         "bootstrapServers": "{{ .Release.Name }}-kafka:9092",
+        {{- else }}
+        {{ $kafka := .Values.hono.adapters.kafkaMessagingSpec.commonClientConfig }}
+        "saslMechanism": "{{ (index $kafka "sasl.mechanism") | lower }}",
```

Comment:
Could you use "plain" as fallback here? The "sasl.mechanism" property might not have been set in `commonClientConfig`.